src/vm/internal/compiler/lex.cc

```diff
@@ -841,10 +846,82 @@ static int skip_to(const char *token, const char *atoken) {
   }
 }
 
+void init_include_path() {
+  svalue_t *ret;
```

Comment:
To use STL I had to rewrite the complete handling of include, something I don't want to try without being able to compile and run the driver on my own, although I have to agree that that is something to do in the future…

Concerning goto:
This (some cleanup in case of errors) is the only case I would allow the usage of goto, since the alternative of code doubling in my eyes is worse and yields easily to mistakes.
The other alternative would be switching to classes, again something I won't try as long as I don't have a 'real' computer available to try those things on my own (my tablet doesn't even have a */bin/sh*! And I can't put one there either 'cause I don't have write permission on */*)…

And for the readability I've added all those comments?!